lib/person_cards.rb

```diff
@@ -39,6 +41,10 @@ def identifiers
     Section::Identifiers.new(person, top_identifiers: top_identifiers).data
   end
 
+  def memberships
+    @person.memberships.select { |m| m.term.id == @term.id }
```

Comment:
Rather than rolling your own `select` here, it's probably better to go through the public search interface: i.e. `@person.memberships.where(legislative_period_id: term.id)`